INSTALL.md

```diff
@@ -120,3 +120,17 @@ REM Install "release with debug information" configuration
 cmake --install . --config relwithdebinfo
 REM Well done! Your installed oneTBB is in %TMP%\my_installed_onetbb
 ```
+
+# Installation from vcpkg
+
+You can download and install oneTBB using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
+```sh
```

Comment:
@pavelkumbrasev Maybe it is worth adding comments to this example as well? As we did for the rest of them